util/integration-test.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/sh
+
+# configuration file location and volume name
+CFG=/tmp/minetest.conf
+MINETEST_VOLUME=minetest
+
+# set proper permissions in the volume
+docker run --rm -i -v ${MINETEST_VOLUME}:/minetest alpine /bin/sh -c "chown -R 30000.30000 /minetest"
```

Comment:
I am 99% sure you can just run `chown -R 30000:30000 ${MINETEST_VOLUME}` from outside

Reply:
Sadly no: the volume is an abstract construct (but it would reside in `/var/lib/docker/volumes` in _this_ case).
I think i'll use a bind-mount instead of the volume, that would make the `chown`-ing less weird